pkgs/applications/kde/angelfish.nix

```diff
@@ -15,13 +14,14 @@
 , kwindowsystem
 , qtfeedback
 , qtquickcontrols2
+, qqc2-desktop-style
 , qtwebengine
 , rustPlatform
 , srcs
 
 # These must be updated in tandem with package updates.
-, cargoShaForVersion ? "23.01.0"
-, cargoSha256 ? "sha256-dIXA875HsG56baHrTWw9L560n4s0wRv6Ag/2oj1x0gk="
+, cargoShaForVersion ? "23.03.80"
+, cargoSha256 ? "sha256-nXWQNXVlPWRQdDPXR5NsHJR6kaLxV7UxPC0OIubH/4c="
```

Comment:
Can we inline them. with finalAttrs such hacks should no longer be necessary

Reply:
I don't see why not, but it'll still probably need some sort of assertion to prevent confusion from updatescripts...